subsys/bluetooth/audio/Kconfig.mcs

```diff
@@ -121,4 +121,100 @@ config BT_MCC_SHELL
 	help
 	  This option enables shell support for the Media Control Client.
 
+config BT_MCC_MINIMAL
+	bool "Minimal Media Control Client without optional procedures"
+	default n
+	help
+	  This option disables all optional procedures in the Media Control Client.
+
+config BT_MCC_READ_MEDIA_PLAYER_ICON_URL
```

Comment:
What I notice looking at the spec is that there is no optional procedure called `Read Media Player Icon URL`, but there is one called `Read Media Player Icon Object Information`. Mentions of "URL" are only in the mandatory `Read Media Information`. I don't know this particular spec well enough, if it is that they refer to the same resource or not. However, since these Kconfigs are making procedures in spec optional, as per spec, I think that the names should correspond to spec to make it easier for the developer.

Reply:
agree with there is no optional procedure called "Read Media Player Icon URL", there option is used to control the characteristics "Media Player Icon URL". I think I will update help to be more clear about config intention.

4.5.1
If the Media Player Icon URL characteristic is exposed by the Media Control Server, the Media Control 
Client may read the characteristic value of the Media Player Icon URL characteristic. The Media Player 
Icon URL characteristic value can be used to obtain the associated icon for this media player via the

Reply:
Agree with @pin-zephyr 

The `Read Media Information procedure` may optionally read the icon URL as well, so it makes sense to optionally support that